firefox-ios/Client/Helpers/MenuBuilderHelper.swift

```diff
@@ -137,7 +156,13 @@ class MenuBuilderHelper {
 
         let viewMenu = UIMenu(options: .displayInline, children: viewMenuChildren)
         viewMenu.children.forEach {
-            ($0 as! UIKeyCommand).wantsPriorityOverSystemBehavior = true
+            guard let viewMenuKeyCommand = $0 as? UIKeyCommand else {
```

Comment:
I'm thinking that we actually can avoid those logs also in the other class, i'd refactor the code to be:
```
($0 as? UIKeyCommand)?.wantsPriorityOverSystemBehavior = true
```
And remove all the logs.